pkg/api/testapi/testapi.go

```diff
@@ -79,6 +79,10 @@ func init() {
 	Extensions = Groups[extensions.GroupName]
 }
 
+func (g TestGroup) ContentConfig() (string, *unversioned.GroupVersion, runtime.Codec) {
+	return "application/json", g.GroupVersion(), g.Codec()
+}
+
```

Comment:
I think I might have missed where this is used and what it's for. These 3 values seem readily available. I would understand if this actually returned a ContentConfig, but it only gives you 3 easily gettable values used to build one.